coalib/bearlib/abstractions/Linter.py

```diff
@@ -569,17 +610,43 @@ def __repr__(self):
             return '<{} linter object (wrapping {!r}) at {}>'.format(
                 type(self).__name__, self.get_executable(), hex(id(self)))
 
+    class LocalLinterBase(LinterBase, LocalBear):
+        pass
+
+    class GlobalLinterBase(LinterBase, GlobalBear):
+
+        @staticmethod
+        def create_arguments(config_file):
+            """
+            Creates the arguments for the linter.
+
+            You can provide additional keyword arguments and defaults. These
+            will be interpreted as required settings that need to be provided
+            through a coafile-section. This is the file agnostic version for
+            global bears.
+
+            :param config_file:
+                The path of the config-file if used. ``None`` if unused.
+            :return:
+                A sequence of arguments to feed the linter-tool with.
+            """
+            raise NotImplementedError
```

Comment:
maybe you should copy the default implementation for `LocalBear` also the `LocalLinterBase`?